autora/css/style.css

```diff
@@ -0,0 +1,397 @@
+* {
+    box-sizing: border-box;
+}
+
+
+html {
+    font-size: 10px; /* definir uma font-size padrão */
+}
+
+
+body {
+    font-family: 'Montserrat', Arial, Helvetica, sans-serif;
+    margin: 0;
+}
+
+
+a {
+    text-decoration: none;
+    color: inherit;
+}
+
+
+ul {
+    padding: 0;
+}
+
+h5{
+    width: 56%;
+    margin: 0 auto;
+}
+
+
+li {
+    list-style: none;
+}
+
+/* define que as imagens tem que ter o tamanho máximo da div mãe */
+.img-responsive {
+    width: 100%;
+}
+
+/* define o tamanho da caixa */
+.container {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 959px;
+}
+
+
+.navegacao {
+    background-color: #674582;
+    color: white;
+    text-transform: uppercase;
+    box-shadow: 0px 5px 5px rgba(0,0,0,.2);
+    padding: 1.5rem;
+    z-index: 99;
+    position: fixed;
+    width: 100%;
+}
+
+.navegacao__logo {
+    max-width: 200px;
+}
+
+
+.nav__box {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+
+.navegacao__itens {
+    position: absolute;
+    right: -700px;
+    top: 85px;
+    width: 100%;
+    background-color: #674582;
+    text-align: center;
+    padding: 3rem 0;
+    transition: right 1s;
+}
+
+
+.navegacao__nav ul{
+    display:flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+
+.navegacao__nav li{
+    margin-right: 10px; 
+    font-size: 1.2rem; 
+}
+
+
+.navegacao__redes-sociais a {
+    margin: 0 1rem;
+}
+
+
+.header {
+    min-height: 80vh;
+    height: 500px;
+    background: linear-gradient(rgba(255, 0, 255, 0.2),
+                                rgba(255, 0, 255, 0.2)),
+                url(../imagens/vivyheader.jpg);
+    background-size: cover;
+    background-position: center 14%;
+    background-repeat: no-repeat;
+    display: flex;
+    align-items: flex-end;
+    text-align: center;
+    padding: 11rem 2rem 5rem 2rem;
+    color: white;
+}
+
+.sobre__autora {
+    text-transform: uppercase;
+    font-size: 2.5rem;
+    font-weight: 900;
+}
+
+h1{
+    font-size: 30px; 
+}
+
+.apresentacao{
+    display: flex;
+    width: 100%;
+    align-items: center;
+    text-align: center; 
+    padding-left: 15%;
+}
+
+.apresentacao__texto{
+    margin-left: -10%;
+    color: #674582;
+}
+
+.apresentacao__autora{
+    font-size: 25px;
+    text-align: left;
+    margin-left: 18%;
+}
+
+
+.img__autora{
+    max-width: 19%;
+    margin:0 auto; 
+    padding: 80px 0;
+}
+
+h4 p{
```

Comment:
vi, acho que esse 'p' tá aqui por engano, né? até pq não pode ter um "p" dentro de um "h4"

Reply:
Pode se o p for filho de h4, o que não é o caso aqui, porém pode